.eslintrc.js

```diff
@@ -66,8 +67,26 @@ module.exports = {
     {
       files: ['test/unit/cloudant.v1.test.js'],
       rules: {
-        'prettier/prettier': 'off',
         'node/no-missing-require': 'off',
+        'prettier/prettier': 'off',
+      },
+    },
+    {
+      files: ['cloudant/v1.ts'],
+      rules: {
+        '@typescript-eslint/no-empty-interface': 'off',
+        '@typescript-eslint/no-explicit-any': 'off',
+        '@typescript-eslint/no-namespace': 'off',
+        '@typescript-eslint/no-unused-vars': 'off',
+        'no-redeclare': 'off',
```

Comment:
the ts pair of `no-redeclare` is missing:
```suggestion
        '@typescript-eslint/no-redeclare': 'off',
        'no-redeclare': 'off',
```

Reply:
I only added the ignores that were necessary. Ie. this should be a minimal PR that passes. I'm hesitant to add ignores that are not necessary right now.